website/src/pages/CRON/CRON.jsx

```diff
@@ -0,0 +1,682 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useState, useEffect } from "react";
+import RingLoader from "react-spinners/RingLoader";
+import { useNavigate } from "react-router-dom";
+import { ResponsiveLine } from "@nivo/line";
+
+import "../CRON/cron.css";
+
+import { errorApi, formatByteForGB } from "../../utils/Utils";
+import CronSummary from "../../components/CRONComponents/CRONSummary/CronSummary";
+
+const CRON = () => {
+  const urlParams = new URLSearchParams(window.location.search);
+  const [dataCronSummary, setDataCronSummary] = useState([]);
+  const [dataCron, setDataCron] = useState([]);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isLoading2, setIsLoading2] = useState(true);
```

Comment:
you should consider using different names for `isLoading`, adding a `2` here is not helpful